tests/testapp/factories.py

```diff
@@ -180,3 +182,20 @@ class Meta:
 class MiddlewareModelFactory(factory.django.DjangoModelFactory):
     class Meta:
         model = MiddlewareModel
+
+
+class RedirectFactory(factory.django.DjangoModelFactory):
+    """
+    TODO: Consider moving to wagtail factories?
+    """
+
+    old_path = factory.Faker("slug")
+    # Note: Site needs `hostname` and `port` to be a unique combination,
+    # so this seems like the best option:
```

Comment:
Let's rm this comment after changing the below to a `SiteFactory`

Reply:
This comment is actually a response to issues arising from using `SiteFactory` there, but I will re-test it.

Reply:
Addressed in 2822f362c4cfc4dfd11385da06ac38d630f1824a